src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -177,14 +177,27 @@ const SystemsExposedTable = (props) => {
         );
     };
 
+    const getSelectedSystemsWithName = () => {
+        if (!selectedHosts) {
+            return [];
+        }
+        // we only need display name when we have 1 system selected
+        else if (selectedHosts.length === 1) {
+            const system = items.data.find(item => item.id === selectedHosts[0]);
```

Comment:
you can apply some ES6 sugar syntax here

```suggestion
            const {id, display_name: displayName} = items.data.find(({ id }) => id === selectedHosts[0]);
            return [{ id, displayName}]; 
```

Be aware to change the variable `display_name` to `displayName` where it's used

Reply:
I tweaked this to keep the `display_name` instead of `displayName` which more consistent with already existing code. Is it OK?
```suggestion
            const {id, display_name} = items.data.find(({ id }) => id === selectedHosts[0]);
            return [{ id, display_name}];
```